server/utils/getInstallationInfo.ts

```diff
@@ -0,0 +1,50 @@
+import env from "@server/env";
+import { version } from "../../package.json";
+import fetch from "./fetch";
+
+const dockerhubLink =
+  "https://hub.docker.com/v2/repositories/outlinewiki/outline";
+
+export function getVersion(): string {
+  return env.VERSION ? env.VERSION : version;
```

Comment:
Don't use `env.VERSION` here, it might be a `sha` or some other non-semver value, the version in `package.json` is the right one to use